browser/src/communication/federated/federated_client.ts

```diff
@@ -119,13 +123,34 @@ export class FederatedClient extends Client {
     }
   }
 
+  private addDifferentialPrivacy (currentRoundWeights: Weights): Weights {
+    const weightsDiff = List(currentRoundWeights)
+      .zip(List(this.previousRoundWeights))
+      .map(([w1, w2]) => w1.add(-w2))
+
+    const normBound = weightsDiff.map((w) => w.norm().dataSync()[0]).max()
```

Comment:
add a comment what kind of norm you compute. why is there a max? norm is 2-norm but at the end gets maxed, like infty-norm?

Reply:
that's an error on my side! will push changes soon